code/modules/uplink/uplink_items.dm

```diff
@@ -2021,6 +2021,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	restricted_roles = list("Assistant")
 	surplus = 0
 
+/datum/uplink_item/role_restricted/theincrediblebulk
+	name = "A Hulk Injector"
+	desc = "A Very Powerful Mutation Discovered in a ancient vault on mars, causes rapid musclegrowth."
```

Comment:
```suggestion
	name = "Hulk Injector"
	desc = "A very powerful mutation discovered in a ancient vault on mars, When applied the mutation causes rapid muscle growth turning the user in to a hulk."
```

name isnt really a suggestion
desc is just fixing of grammatical errors and making it have some fluff to it